registry/handlers/health_test.go

```diff
@@ -0,0 +1,200 @@
+package handlers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/docker/distribution/configuration"
+	"github.com/docker/distribution/health"
+	"golang.org/x/net/context"
```

Comment:
Several existing tests use `golang.org/x/net/context`. Should I update these while I'm at it?